webpack.config.js

```diff
@@ -57,7 +61,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
             rules: [
                 {
                     test: /\.js$/,
-                    exclude: /node_modules/,
+                    exclude: /node_modules\/(?!(sulu-(.*)-bundle|@ckeditor|lodash-es)\/)/,
```

Comment:
Why is that suddenly necessary?

Reply:
yes it is, its the same regex as used in transformIgnorePatterns.

Reply:
I asked why it is suddenly necessary... Because it seems like it was not necessary before (the build was working without these lines... Is it because we are including the files by their package name instead of their path now?

Reply:
yes because of that impports, the other (ckeditor, lodash) I added as the regex in the package.json did that also.